src/Services/cdl/LtInfo.js

```
@@ -1,67 +1,44 @@
      * @param {String} filePath File this logtype belongs to.
      */
-    constructor (ltInfoSST, fid, filePath) {
-        this.lt = ltInfoSST;
-        this.lt.fid = fid;
-        this.lt.filePath = filePath;
-        this.lt.fileName = filePath.split("/").pop();
-        this.childIds = [];
+    constructor (ltInfo, filePath) {
+        for (var key in ltInfo) {
+            this[key] = ltInfo[key];
+        }
```

ESLint

[error] The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.

(guard-for-in)